src/views/project/project/modules/project-alert/ProjectEscalationPolicy.vue

```diff
@@ -86,17 +120,35 @@ export default {
             escalationPolicyName: computed(() => get(state.escalationPolicyRule, 'name')),
             finishCondition: computed(() => {
                 const finishCondition = get(state.escalationPolicyRule, 'finish_condition');
-                if (finishCondition) return finishCondition.label;
+                if (finishCondition) return state.finishConditions.find(d => d.name === finishCondition).label;
                 return '';
             }),
             escalationRules: computed(() => get(state.escalationPolicyRule, 'rules')),
             repeatCount: computed(() => get(state.escalationPolicyRule, 'repeat_count')),
+            //
+            projectChannels: [],
+            projectChannelForLevel: {},
         });
 
         /* util */
         const notificationLevelFormatter = str => str.replace('LV', '');
+        const channelFormatter = level => filter(state.projectChannels, { notification_level: level });
 
         /* api */
+        const apiQuery = new ApiQueryHelper();
+        const getQuery = () => {
+            apiQuery
+                .setFilters([{ k: 'project_id', v: props.projectId, o: '=' }]);
+            return apiQuery.data;
+        };
+        const listProjectChannel = async () => {
+            try {
+                const { results } = await SpaceConnector.client.notification.projectChannel.list({ query: getQuery() });
+                state.projectChannels = results;
+            } catch (e) {
+                console.error(e);
```

Comment:
```suggestion
                state.projectChannels = [];
                console.error(e);
```
캐치에 초기화 로직 넣으면 더 좋을거같습니당